collect_app/src/main/java/org/odk/collect/android/formentry/audit/AuditUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.odk.collect.android.formentry.audit;
+
+import org.javarosa.form.api.FormEntryPrompt;
+import org.odk.collect.android.logic.FormController;
+
+import static org.javarosa.form.api.FormEntryController.EVENT_GROUP;
+import static org.javarosa.form.api.FormEntryController.EVENT_QUESTION;
+import static org.javarosa.form.api.FormEntryController.EVENT_REPEAT;
+
+public class AuditUtils {
+
+    private AuditUtils() {
+
+    }
+
+    public static void logCurrentEvent(FormController formController, AuditEventLogger auditEventLogger, long currentTime) {
```

Comment:
This is more like `logCurrentScreen` or `logCurrentEvents`, I think?

Reply:
In my head it's logging the "current event" of the `FormController` but I see  now that is confusing. I'll make that explicit.